.github/workflows/main.yml

```diff
@@ -39,13 +39,14 @@ jobs:
           fetch-depth: 0
       - uses: ./.github/setup
 
-      - name: Test
-        run: pnpm turbo --filter tests test-ci
+      - name: Build tRPC, then test
+        run: pnpm turbo --filter "@trpc/*" build
+      - run: pnpm vitest --coverage
 
-      - uses: codecov/codecov-action@v4
-        with:
-          # token: ${{ secrets.CODECOV_TOKEN }} # not required for public repos
-          directory: ./packages
+      - uses: codecov/codecov-action@v3
```

Comment:
REVERT ME

v4 requires a token: 
![image](https://github.com/trpc/trpc/assets/51714798/544f7b9c-6af5-4a1c-86f7-53800bd6a4ea)

Reply:
We have a `CODECOV_TOKEN` in the repo already that should work